builds/org.eclipse.n4js.product.build/pom.xml

```diff
@@ -28,6 +28,41 @@ Contributors:
 	</scm>
 
 	<profiles>
+		<profile>
+			<id>stagingNpmRegistry</id>
+			<activation>
+				<property>
+					<name>env.STAGING_NPM_REGISTRY</name>
+				</property>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>exec-maven-plugin</artifactId>
+						<version>${codehaus-exec-maven-plugin.version}</version>
+						<executions>
+							<!-- Publish n4js-libs to staging area when the env variable STAGING_NPM_REGISTRY is set -->
```

Comment:
better `if` instead of `when`?
Also: add more detail. Is this the publication to the verdaccio only intended for testing?

Reply:
done